lib/unparser.py

```diff
@@ -8,8 +8,8 @@
 # create a simplified function by extracting only the AST nodes used
 
 """Usage (for testing, will do roundtrips on the files):
-  unparse.py <path to source files>
-  unparse.py --testdir <path to source files directories>
+unparse.py <path to source files>
+unparse.py --testdir <path to source files directories>
```

Comment:
Same comment as in the other files above regarding adding indentation to the `Usage` section content.

Reply:
I did it but black changed this back...